src/fitnesse/components/ComponentFactory.java

```diff
@@ -24,10 +24,16 @@
 public class ComponentFactory {
 
   private final Properties properties;
+  private final ClassLoader classLoader;
   private Map<String, Object> components;
 
   public ComponentFactory(Properties properties) {
+    this(properties, ClassLoader.getSystemClassLoader());
```

Comment:
Shouldn't this get its default class loader from `ClassUtils`?
Actually, if this class needs/has its own class loader reference it should probably not have a constructor that does not supply a class loader.

Reply:
There are a few places deep in the FitNesse code that do Class.forName magic.

Preferable I want to get rid of the ClassUtils class, since it's introducing a static variable. For now I want to leave it like this, because working the ClassUtils.forName out of those pieces of code is a huge endeavour. I should probably put some documentation describing my intent on the `ClassUtils` class.